securedrop_export/export.py

```diff
@@ -173,7 +202,7 @@ def check_usb_connected(self):
                 stderr=subprocess.PIPE)
             self.exit_gracefully("USB_CONNECTED")
         except subprocess.CalledProcessError:
-            self.exit_gracefully("USB_NOT_CONNECTED")
+            self.exit_gracefully(ExportStatus.USB_NOT_CONNECTED.value)
```

Comment:
why not use `safe_check_call` here?

Reply:
if you are referring to https://github.com/freedomofpress/securedrop-export/pull/28/files#diff-f7920fa15cd44ca4817dafa773cf5372R200 specifically, the reason is that is `check_output` and we would need to create a new wrapper method for it. Given it was used only once, it was not worth the maintenance cost. What do you think?

Reply:
oh i see, the wrapper is for `check_call` not `check_output`. looks like we have another call to `check_output` when getting the printer uri so a new wrapper for `check_output` might be a good idea, but I'm fine with delaying until after we finish printer support in the client.